src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+//spread 
+//TimeStamp has time prop
+const ChatEntry = ({ updateChat, ...message}) => {
+
+  const onLikeClick = () => {
+    updateChat({ ...message, liked: !message.liked });
+  };
+
 
-const ChatEntry = (props) => {
   return (
     <div className="chat-entry local">
```

Comment:
The optional alignment feature would require us to be able to pick whether to use the `local` or `remote` class here. If the `App` made note of the sender of the first message, and passed that down as a prop, such as `localSender`, then we could use that value and the value of the current message's `sender` to determine whether this was a `local` or `remote` message.